examples/tcp_connection/lib/tcp_connection.ex

```diff
@@ -40,7 +40,7 @@ defmodule TCPConnection do
   end
 
   def connect(opts) do
-    host        = Keyword.fetch!(opts, :hostname) |> String.to_char_list()
+    host        = Keyword.fetch!(opts, :hostname) |> String.to_charlist()
```

Comment:
Should bump the elixir requirement in the mix.exs as this is not in 1.0.

Reply:
@fishcakez  Should I bump it to elixir 1.6.2 on both examples